tests/udf/available_python_packages_utils_test.py

```diff
@@ -0,0 +1,30 @@
+from exasol_python_test_framework import udf, docker_db_environment
+from exasol_python_test_framework.udf.available_python_packages_utils import run_python_package_import_test
+
+import unittest
+
+class AvailablePackageTest(udf.TestCase):
+    SCHEMA = "AvailablePackageTest"
+
+    def setUp(self):
+        self.query(f'CREATE SCHEMA {self.SCHEMA}', ignore_errors=True)
+        self.query(f'OPEN SCHEMA {self.SCHEMA}', ignore_errors=True)
+
+    def test_package_available(self):
+        run_python_package_import_test(self, self.SCHEMA, "PYTHON3", "pandas", fail=False, alternative=None)
+
+    def test_package_not_available_expected(self):
+        run_python_package_import_test(self, self.SCHEMA, "PYTHON3", "asdgkdbalgf", fail=True, alternative=None)
```

Comment:
Maybe use more descriptive name for the package:
```suggestion
        run_python_package_import_test(self, self.SCHEMA, "PYTHON3", "NOT_INSTALLED_PACKAGE", fail=True, alternative=None)
```
```suggestion
        run_python_package_import_test(self, self.SCHEMA, "PYTHON3", "asdgkdbalgf", fail=True, alternative=None)
```